src/Jupyter/Visualization/ExecutionPathEncoder.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.Jupyter.Core;
+using Microsoft.Quantum.IQSharp.Core.ExecutionPathTracer;
+
+namespace Microsoft.Quantum.IQSharp.Jupyter
+{
+    /// <summary>
+    ///     Contains the ID of the div used to populate the
+    ///     <see cref="ExecutionPath"/> visualization.
+    /// </summary>
+    public class ExecutionPathDisplayable
```

Comment:
Would it be worth generalizing this so that it can be used by other IQ# features that need to output a `<div>` with a known `id` and `class` attributes? Might be good to work with @Reemeela2121 to come up with a common solution.

Reply:
Yea, that would be a good idea. Maybe a `DisplayableDiv` class?

Reply:
That could work, yes. My concern would be that the division between displayable objects and result encoders is intended to keep presentation logic out of displayable objects as much as possible, such that including "Div" in the name would break that separation. That said, this is a weird case in that there's no meaning whatsoever to "printing a `<div>` element" when considering any MIME type other than text/html...

Reply:
cc-ing @anpaz-msft @rmshaffer in case they have thoughts on the best way to accomplish rendering an HTML string

Reply:
One idea: rename this class to something like `DisplayableHtmlElement` and the encoder to `DisplayableHtmlElementEncoder`? Then it's clear that it only makes sense from HTML.